README.md

```diff
@@ -143,6 +143,44 @@ guard condition else { return }
 if condition { } else { }
 ```
 
+### Commas
+
+There should be no space before an comma and a single one after any comma.
+
+**Preferred:**
+```swift
+function print(name: String, surname: String) {
+  ...
+}
+```
+
+**Not Preferred:**
+```swift
+function print(name: String ,surname: String) {
+  ...
+}
+```
+
+## Parentheses
+
+Parentheses around conditionals, if,for,while,do statements, are not required and should be omitted.
```

Comment:
For consistency with the rest of the documentation, add a space after the commas.